src/main/scala/org/dia/algorithms/mcc/MCSUtils.scala

```diff
@@ -0,0 +1,206 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dia.algorithms.mcc
+
+import java.util
+import scala.collection.mutable.ListBuffer
+import ucar.ma2.{ArrayInt, ArrayDouble, DataType}
+import ucar.nc2.{NetcdfFileWriter, Dimension, Attribute, Variable }
+import scala.reflect.ClassTag
+import scala.collection.JavaConversions._
+import scala.collection.mutable
+
+import org.dia.core.SciTensor
+import org.dia.tensors.AbstractTensor
+
```

Comment:
The style checker checks the order of import statements.
Each import group is line separated.

Here is the template :

import java...

import scala....

import everything that's not spark or scispark

import org.apache.spark

import org.dia.... <-- us

Imports within a group follow alphabetical ordering.